setup/setupEnvironment.sh

```diff
@@ -238,6 +238,20 @@ cat << EOF >> ${DELETE_SCRIPT}
 echo "Deleting API Gateway"
 aws apigateway delete-rest-api --rest-api-id ${GATEWAY_ID}
 
+echo "Deleting Cloudwatch logs for the Lambda functions"
```

Comment:
but definitely a good idea...something like this would work though: /aws/lambda/rekognition-add-picture-20170225112145 (and all of the info is in the script to generate, if you want, otherwise I'll work on that later).